File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1038.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');
    
a = parseFloat(lines.shift());
b = parseFloat(lines.shift());

if(a==1){
    console.log("Total: R$", (b*4).toFixed(2));
}
else if(a==2){
    console.log("Total: R$", (b*4.5).toFixed(2));
}
else if(a==3){
    console.log("Total: R$", (b*5).toFixed(2));
}
else if(a==4){
    console.log("Total: R$", (b*2).toFixed(2));
}
else{
    console.log("Total: R$", (b*1.5).toFixed(2));
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1038.py
n = [float(i) for i in input().split(' ')]
a = n[0]
b = n[1]

if(a==1):
    print("Total: R$ {:.2f}".format(b*4.0))
elif(a==2):
    print("Total: R$ {:.2f}".format(b*4.5))
elif(a==3):
    print("Total: R$ {:.2f}".format(b*5.0))
elif(a==4):
    print("Total: R$ {:.2f}".format(b*2.0))
elif(a==5):
    print("Total: R$ {:.2f}".format(b*1.5))